packages/issuer-rest/lib/api.ts

```diff
@@ -75,6 +72,8 @@ export class RestAPI {
 
   constructor(opts?: { metadata: IssuerMetadata; stateManager: ICredentialOfferStateManager; userPinRequired: boolean }) {
     dotenv.config()
+    const key = fs.readFileSync(process.env.PRIVATE_KEY || path.join(__dirname, './privkey.pem'), 'utf-8')
```

Comment:
Why are you adding the notion of keys and certificates to the issuer, which should be completely neutral towards keys?

Reply:
One of the requirements of PAR is to use https and that's the reason of having a private key and certificate there. It's not passed to the issuer at any point, only to the web server. It must be called after dotenv.config() otherwise the environment variable is not picked up. Unless you are asking me to remove the private key and certificate from the .env file.

Reply:
So are we talking about a client certificate here or a server certificate? If the latter it doesn't make sense that the code has any notion of that. If the former, the names are really confusing

Reply:
TLS termination is not the concern of the API/Server. So it simply is out of scope for the code.

If you need it you use a reverse proxy or loadbalancer with certificate. I rather see an option where we allow PAR to not use https in case we are in development mode. So by default it is required and only if this option is enabled we are okay with plain http (which should not be used in production of course)

Reply:
Removed https implementation/support from api.ts